python/metric_pipeline_utils/util.py

```diff
@@ -0,0 +1,636 @@
+# LSST Data Management System
```

Comment:
Do we really need to port over all the utils code?  I thought part of this process was to bring over what we needed, but no more.  If we bring this code over as a monolithic thing, we end up with a bunch of untested code, or we need to write tests for it all.